src/components/index/Card.tsx

```diff
@@ -0,0 +1,46 @@
+import { clsx } from "clsx";
+import Image from "next/image";
+
+interface CardProps {
+  readonly title: string;
+  readonly description: string;
+  readonly imageSrc?: string;
+  readonly className?: string;
+  readonly isPriority?: boolean;
+}
+
+export function Card({
```

Comment:
I think we should stick with component structure described in [CONTRIBUTING.md](https://github.com/Bartek532/rssmarkable/blob/main/CONTRIBUTING.md#example-component-structure)